composer.json

```diff
@@ -16,7 +16,7 @@
         "php-http/client-common": "^1.9 || ^2.0"
     },
     "require-dev": {
-        "symfony/phpunit-bridge": "^5.3",
+        "symfony/phpunit-bridge": "^5.4.33 || ~6.3.10 || ^6.4.1",
```

Comment:
do we need to allow 6.3.10? 

as this is only require-dev, i'd like to keep the list as short as possible.

Reply:
in fact, using `^6.4.1` is enough